Makefile

```diff
@@ -196,8 +196,8 @@ $(BIN)/calico $(BIN)/calico-ipam: $(SRCFILES) vendor
 	-w /go/src/$(PACKAGE_NAME) \
 	-e GOCACHE=/go-cache \
 		$(CALICO_BUILD) sh -c '\
-			go build -v -o $(BIN)/calico -ldflags "-X main.VERSION=$(GIT_VERSION) -s -w" calico.go ; \
-            go build -v -o $(BIN)/calico-ipam -ldflags "-X main.VERSION=$(GIT_VERSION) -s -w" ipam/calico-ipam.go'
+			go build -v -o $(BIN)/calico -ldflags "-X github.com/projectcalico/cni-plugin/pkg/cniplugin.VERSION=$(GIT_VERSION) -s -w" ./cmd/calico && \
```

Comment:
I think in our other projects we keep the VERSION as part of the main package. 

https://github.com/projectcalico/felix/blob/master/cmd/calico-felix/calico-felix.go#L40
https://github.com/projectcalico/node/blob/master/cmd/calico-node/main.go#L88

Reply:
I think those a both counter-examples :-P but I do prefer `-X main.VERSION` over the convoluted package name (and make it easier to build from the other package) so making that change.

I don't think it makes sense to move argument parsing into the main package for now; we  can always factor that out later (splitting `Main()` into `Main()`, which does arg parsing and a second method that skips it, if it becomes useful.

Reply:
Hah, yeah you're right they are! Sigh...